docs/10.0/guides/integrate-with-react/react-redux-example.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: 'React: Redux example'
-metaTitle: 'React: Redux example - Guide - Handsontable Documentation'
+title: 'Redux example'
```

Comment:
Hey @kirszenbaum I see that `React: Redux example` is replaced by `Redux` example. I assume that it did not switch to `Redux example in React` as Redux is used only in React so it's not needed to add that note.

Reply:
nvm, I saw that you did the same with Vuex so it answers by question